platforms/hyperledger-fabric/charts/peernode/values.yaml

```diff
@@ -167,3 +167,13 @@ proxy:
   #Eg. port: 8443
   port: 8443
   
+config:
+  # Set memory limits of pod  
+  pod:  
+    resources:
+      # Provide the limit memory for node
+      # Eg. limits: 512M
+      limits:
+      # Provide the requests memory for node
+      # Eg. requests: 550M
+      requests:
```

Comment:
[nitpick] No new line at the end of this file.

Reply:
fixed